### DISCUSSION TITLE
I cannot find anything on how to make an api request with fetch.

### DISCUSSION THREAD

**Discussion by zachfran** [supabase/supabase#2604] [Questions] [answered]
When ever I actually find something and try doing a api request with fetch, nothing happens at all. It just errors. Please lmk if theres docs to making a POST request with fetch. ;D

**Comment by silentworks** [comment, marked as answer]
Can you post the code example you are using to try and make this request please?

**Reply by zachfran** [reply]
Figured out my problem.

**Reply by silentworks** [reply]
It will probably be useful for someone else if you still shared your code example along with the solution you came up with.
